packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/manageAccounts.tsx

```diff
@@ -29,6 +29,11 @@ import { Account, selectAccounts, setActiveAccount } from 'store/account'
 import { selectActiveAccount } from 'store/account'
 import { selectIsPrivacyModeEnabled } from 'store/settings/securityPrivacy'
 import { selectActiveWalletId, selectWallets } from 'store/wallet/slice'
+import { WalletType } from 'services/wallet/types'
+import { CoreAccountType } from '@avalabs/types'
+
+const VIRTUAL_WALLET_ID = 'private-key-accounts'
+const VIRTUAL_WALLET_NAME = 'Imported'
```

Comment:
```suggestion
const IMPORTED_ACCOUNTS_VIRTUAL_WALLET_ID = 'imported-accounts-wallet-id'
const IMPORTED_ACCOUNTS_VIRTUAL_WALLET_NAME = 'Imported'
```
There might be other types of imported accounts in the future besides private key accounts.

Reply:
Changed in IMPORTED_ACCOUNTS_VIRTUAL_WALLET_ID